src/clims/api/endpoints/substance_file.py

```diff
@@ -8,16 +8,34 @@
 from rest_framework import status
 from django.db import IntegrityError
 
+from sentry.api.paginator import OffsetPaginator
 from sentry.api.bases.organization import OrganizationEndpoint
-from clims.services import RequiredHandlerNotFound, FileNameValidationError
+from clims.services import FileNameValidationError
+from clims.handlers import RequiredHandlerNotFound
+from clims.api.serializers.models.organization_file import OrganizationFileSerializer
 
 
 class SubstanceFileEndpoint(OrganizationEndpoint):
     permission_classes = (IsAuthenticated, )
 
     def get(self, request, organization):
-        # TODO: and write test
-        return Response({})
+        """
+        List all substance submission files in the system
+        `````````````````````````````````````````````````
+
+        :pparam string organization_slug: the slug of the organization the
```

Comment:
Typo?

Reply:
Do you mean pparam?

Reply:
Yes.

Reply:
That is used in the codebase. They parse it here: src/sentry/utils/apidocs.py

This is a sentry thing. These are "path" parameters used in the api endpoint documentation. Look for "path parameter" here:

https://docs.sentry.io/api/events/get-project-group-index/

Reply:
Aha! Got it!